configure.ac

```diff
@@ -514,9 +514,7 @@ AS_IF([test x$have_check = x], [
 ])
 
 PKG_CHECK_MODULES([CAP], [libcap], [have_libcap=1], [have_libcap=])
-AS_IF([test x$have_libcap = x], [
-    AC_MSG_ERROR([libcap is missing])
-], [
+AS_IF([test x$have_libcap = x], [], [
```

Comment:
My worry is that if a library is actually available but missing in the build env, then SSSD will be silently built without capabilities support.

At the very least it warrants a loud warning (but who reads those if build doesn't fail...)

For a time being, would it be possible to hard-condition build-without-libcap by BSD platform only?

Reply:
Maybe we should rather make it an enabled-by-default build option? So that a user would first get a configure error and then may decide if he wants to disable capabilities or try fixing his build env.

Reply:
This also works for me.